### DISCUSSION TITLE
How far to get into the assignment?

### DISCUSSION THREAD

**Discussion by JKinsler** [earthlab-education/Earth-Analytics-2023-01-Intro#374] [Q&A04 Raster Data] [answered]
Hi, can you please remind me if the expectation is that we'll complete this whole assignment by the end of the week? Or is the the multi-week assignmet?

**Comment by ellenalamont17** [comment, marked as answer]
It's a multi-week assignment as I recall. I believe Nate said to get as far as the data download, but someone else please confirm that...

**Reply by peterkobylarz** [reply]
This was my recollection as well - get to the point where you download the data.
